modules/core/include/opencv2/core/persistence.hpp

```diff
@@ -1345,6 +1346,19 @@ inline String::String(const FileNode& fn): cstr_(0), len_(0) { read(fn, *this, *
 
 //! @endcond
 
+/** @brief Loads an object from a filestorage.
+
+This performs the same functionality as @cvLoad, but does it directly from a FileStorage object.
+@param filename File storage object
```

Comment:
This parameter should be named `filestorage`. Also could you please remove at sign before`cvLoad` in previous line? It looks like doxygen considered it as internal command.